app/views/GermlineView/components/Board/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect, useState } from 'react';
+
+import ApiPaginatedTable from './components/ApiPaginatedTable';
+import api from '../../../../services/api';
+import columnDefs from './columnDefs';
+import ParamsContext from './components/ParamsContext';
+
+const Board = (): JSX.Element => {
+  const [reports, setReports] = useState([]);
```

Comment:
making a note here that this is typed any, but reports have a way complicated structure right?

Reply:
It's not too complicated for germline reports so I'll add it in